docs/fans.md

```diff
@@ -0,0 +1,36 @@
+﻿## ** What is Shenanigan? ** ![woman_cartwheeling](https://lh6.googleusercontent.com/lJUzNtQP47lexQBdxruB084UG0G2IzRM_rN1tdIaUXNGQF4PMhT0che_hmY8GbxH6q2_Qf9bl4pzT4RXfaLH7iNquGHhv1VfcIWTKEJnCU_rbURBRts0_EGaG56GNaW5yIg9OPEG)
+
+Shenanigan is an open source platform created for athletes and by athletes. Wait whats that, you're not an athlete you say? Well have no fear SHE has a place for you as well. Shenanigan gives athletes a platform to challenge themselves and interact with their biggest fans directly.  SHE gives fans a place to interact with like minds and the athlete themselves through "Fan clubs." Fan's will also have the chance to earn rewards while they watch their favorite athletes push themselves to the limits. Fans can collect rare digital collectibles commemorating those special moments as well as digital version of the athlete themselves! Athlete's can set specific challenges which they must strive to achieve *i.e. land a 5 punch combo or complete a kickflip.* Fan's can then vote on these and have a say on whether or not the athlete failed or succeed! Get out there and meet your favorite athlete, SHE is glad to introduce you!
+
+## ** Why does SHE do it? **![trophy](https://lh3.googleusercontent.com/GHhpehfDWMyYfynU_GNYPlwimDWV3QazaqU9raCCvK0wstXGhMvy5HXBNbWRrNQUyZoAsXrA_FAxSrPFq0mMYAl76Nco4WtJNtNWP11jDWcZRZYmqsoZhhIHorlkr6SfrCRhVPIH)
+
+** Fan ** = "A person who has a strong interest in or admiration for a particular person or thing.”
+
+Now that you know the definition of a fan, internalize it, and ask yourself, what would I do to meet my favorite athlete? How incredible it would be to have specialized content from them? Image the feeling of having your favorite athlete call out your name right before landing a perfect dive. Then having that video moment attached to a digital collectible card forever!  Shenanigan encourages supporting your athlete through donations! It gives athletes the chance to pursue their dreams and fans the chance to earn, while helping and interacting on a more personal level! Shenanigan provides each fan with the web3 instruments necessary to achieve this. Now get out there and grow with your athletes! 
+
+## ** How do you do it? ** ![iphone](https://lh6.googleusercontent.com/PUZX7_Li0FoaCsFyqfU1qve0nbcAQO1lBtNWdltyEyWKaB_RhCyYeJsYIBBJguKMf1qo9kYNdx2c1eLZoBlCojaNMeO1ShWsG93PbiFHf2wC5FuBsvKvTqdVWzs0QLmlwO09Q1ik)
+
+Our fans download the "Shenanigan Mobile App" and register for a fan account. Once registered as a fan the user will interact with the easy to use app interface. User's search for their favorite athlete and once found they can join the athlete's fan club. Once joined users can interact with the community and their favorite athletes! The more a user interacts with their athletes of choice and corresponding fan club's the higher the chance of earning rewards and finding rare digital collectibles known as an “NRG Challenge cards.”
```

Comment:
> NRG Challenge cards

NRG Cards and Challenge cards are different. Challenge Cards are each individual attempt, while NRG Cards hold all the challenge cards inside them, kinda like a pack of cards. Stay tuned this week for the blog post about NRG Cards.